GDJS/Runtime/websocket-debugger-client/websocket-debugger-client.js

```diff
@@ -271,8 +321,22 @@ gdjs.WebsocketDebuggerClient.prototype.sendProfilerOutput = function(
   );
 };
 
-// This is an alternative to JSON.stringify that ensure that circular reference
-// are replaced by a placeholder.
+/**
+ * A Function used to replace Cycles with a default name.
+ * @callback cycleReplacer
+ * @param {string | number} key - The key corresponding to the value.
+ * @param {any} value - The value.
+ */
+
+/** 
+ * This is an alternative to JSON.stringify that ensure that circular reference
+ * are replaced by a placeholder.
+ * @param {any} obj - The object to serialize.
```

Comment:
No need for -

Reply:
The - Is ignored by JSDoc and permits a clearer view of what is the arguments name and what is it's description while browsing through the code.

Reply:
Ok I don't have a strong feeling on this then, we can keep it!